File: logicaFor.js
var Edad=0,EstadoCivil=0,Estatura=0,Sexo=0,PromedioEstatura=0,PorcentajePersonas=0,ContadorPersonas=0, PromedioEdad = 0,
AcomuladorEdad=0,AcomuladorEstatura=0,CantidadPersonas=2; 

for (let numeroPersona = 1; numeroPersona <= CantidadPersonas; numeroPersona++) {
    
    Edad=parseFloat(prompt("Ingrese la edad de la persona "+ numeroPersona));

    Estatura=parseFloat(prompt("Ingrese la estura de la persona"+ numeroPersona+" en cms "));

    EstadoCivil=parseFloat(prompt("Ingrese el Estado Civil de la persona"+ numeroPersona+ " 1.Soltero 2.Casado "));

    Sexo=parseFloat(prompt("Ingrese el sexo de la persona"+ numeroPersona + " 1 Mujer 2.Hombre"));


if((Edad>=18)&&(EstadoCivil==1)&&(Estatura>170)&&(Sexo==1)) {

            ContadorPersonas = ContadorPersonas + 1;
            AcomuladorEdad = AcomuladorEdad + Edad;
            AcomuladorEstatura = AcomuladorEstatura + Estatura;
            }
        }``
        
        PromedioEdad = AcomuladorEdad / ContadorPersonas;
        PromedioEstatura = AcomuladorEstatura / ContadorPersonas;
        PorcentajePersonas = (ContadorPersonas/CantidadPersonas)*100;

        alert("El promedio de edad de las personas que cumplen es: " + PromedioEdad + " años ");
        alert("El promedio de estatura de las personas que cumplen es: " + PromedioEstatura + "centimetros");
        alert("El porcentaje de personas que cumplen con la condición es del: " + PorcentajePersonas + "%");

File: LogicaFor.java
public class LogicaFor{
    int edad = 0, estadoCivil = 0, estatura = 0, sexo = 0, contadorPersonas, acomuladorEdad = 0, acomuladorEstatura = 0,  numeroPersona = 1;
    int cantidadPersonas;
    double promedioEdad, promedioEstatura, porcentajePersonas;
    

public LogicaFor(int edad, int estadoCivil, int estatura, int sexo, int contadorPersonas, int acomuladorEdad,
                int acomuladorEstatura, int numeroPersona, int cantidadPersonas) {
        this.edad = edad;
        this.estadoCivil = estadoCivil;
        this.estatura = estatura;
        this.sexo = sexo;
        this.contadorPersonas = contadorPersonas;
        this.acomuladorEdad = acomuladorEdad;
        this.acomuladorEstatura = acomuladorEstatura;
        this.numeroPersona = numeroPersona;
        this.cantidadPersonas = cantidadPersonas;
}



public void calcularPromedio(){
    
    promedioEdad = acomuladorEdad / contadorPersonas;
    promedioEstatura = acomuladorEstatura / contadorPersonas;
    porcentajePersonas = (contadorPersonas/cantidadPersonas)*100;
    
    System.out.println("El promedio de edad de las personas que cumplen es: " + promedioEdad + " años ");
    System.out.println("El promedio de estatura de las personas que cumplen es: " + promedioEstatura + " centimetros ");
    System.out.println("El porcentaje de personas que cumplen con la condición es del: " + porcentajePersonas + " % ");
     
    
     }
    }